worker/utils/helper.go

```diff
@@ -0,0 +1,67 @@
+package utils
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+
+	"go.uber.org/cadence/client"
+	"go.uber.org/cadence/workflow"
+
+	"worker/config"
+	"worker/models"
+)
+
+func MakeHttpReq[E interface{}](payloadBytes []byte, url string) (E, error) {
+	var successResponse E
+	var errResponse models.ErrorResponse
+
+	req, err := http.NewRequest("POST", url, bytes.NewBuffer(payloadBytes))
+	if err != nil {
+		return successResponse, err
+	}
```

Comment:
might as well pass request `method` as well in the method.
Or maybe define multiple functions for each:
1. MakeGETRequest
2. MakePOSTRequest